courses/migrations/0029_add_certificate_available_date.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.2.15 on 2022-12-19 13:39
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("courses", "0028_alter_certificate_models_make_page_revision_blankable"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="courserun",
+            name="certificate_available_date",
+            field=models.DateTimeField(
+                blank=True,
+                db_index=True,
+                help_text="The day certificates should be available to users. This value is synced automatically with edX studio.",
+                null=True,
```

Comment:
If we set `certificate_available_date` to null, does that mean the certificates will never be generated?

Could we set it to the same value as the course end date, in the migration?

Reply:
>If we set certificate_available_date to null, does that mean the certificates will never be generated?

Yes, just like run's `end_date` it would be like the `certificate_available_date` is open-ended and hasn't arrived yet.

>Could we set it to the same value as the course end date, in the migration?

Yes, we can. I can do that change tomorrow.